src/find/matchers/mod.rs

```diff
@@ -339,6 +340,19 @@ fn parse_str_to_newer_args(input: &str) -> Option<(String, String)> {
     }
 }
 
+/// Creates a file if it doesn't exist.
+/// If it does exist, it will be overwritten.
+fn get_or_create_file(path: &str) -> Result<File, Box<dyn Error>> {
```

Comment:
Would it be better to give the user a more friendly prompt here? But yes, it is better to use "?" for now.
I would like to keep `get_or_create_file` function first, so that it is convenient to modify these unexpected behaviors in a unified way later. :)
Changes have been committed at https://github.com/uutils/findutils/pull/421/commits/d02b2c491f87f00610c62b662c2b13a2ac83b319. thanks.